Quantum-Control-Applications-QuAM/Superconducting/quam_experiments/experiments/qubit_spectroscopy/node.py

```diff
@@ -1,7 +1,7 @@
 from typing import List, Union
 
-from quam_libs.components.superconducting.qubit import AnyTransmon
-from experiments.time_of_flight.parameters import Parameters
+from quam_builder.architecture.superconducting.qubit import AnyTransmon
+from quam_experiments.experiments.time_of_flight.parameters import Parameters
 
 
 def get_optional_pulse_duration(qubits: List[AnyTransmon], node_parameters: Parameters):
```

Comment:
This seems like a common function, does it make sense to have it within qubit_spectroscopy? Wouldn't other nodes need it?